src/Compute/Compute/Generated/Models/PSGalleryImageVersion.cs

```diff
@@ -40,7 +40,8 @@ public string ResourceGroupName
             }
         }
 
-        public GalleryImageVersionPublishingProfile PublishingProfile { get; set; }
+        // This is commented out for hiding the breaking change caused by the client library update.
+        //public GalleryImageVersionPublishingProfile PublishingProfile { get; set; }
```

Comment:
@hyonholee should we just remove these lines since you're adding the property back in the other partial class file for this type?